src/sentry/models/auditlogentry.py

```diff
@@ -60,6 +62,20 @@ def save(self, *args, **kwargs):
 
     def get_actor_name(self):
         if self.actor:
+            # fix display name if needed
+            uuid_regex = re.compile(
```

Comment:
can we split this logic out into helper functions? something like

`is_scim_token_actor` and `format_scim_token_actor_name`

also -- will want to check the scim prefix here as well.